pkg/log/log_test.go

```diff
@@ -20,4 +21,16 @@ func TestSecrets(t *testing.T) {
 		Entry().Infof("My secret is %s.", secret)
 		assert.NotContains(t, buffer.String(), secret)
 	})
+
+	t.Run("should log message only", func(t *testing.T) {
+
+		SetFormatter(logFormatPlain)
+
+		message := "This is a log message"
+
+		var buffer bytes.Buffer
+		Entry().Logger.SetOutput(&buffer)
```

Comment:
I am not sure about it, but you may have to undo this change in order not affect other tests. I.e.:

outWriter := Entry().Logger.Out
defer func() { Entry().Logger.SetOutput(outWriter) }()
var buffer bytes.Buffer
Entry().Logger.SetOutput(&buffer)